### PR TITLE
feat(localization): update translation

### PR DESCRIPTION
**Pull request by jona7150**
Updated translation using Jenkins. Please review and merge @sor10874 

 CC: @michi6571 @masa6674

### CODE REVIEW

File: toolkit/authentication/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -15,11 +15,11 @@
...
   ~
   --><resources>
-    <string name="server_trust_message">試_The certificate provided by %1$s is not signed by a trusted authority____________________________________驗.</string>
-    <string name="cancel">試_Cancel_____________驗</string>
-    <string name="allow_connection">試_Dangerous: Allow Connection____________________________驗</string>
-    <string name="username_password_login_message">試_Please login to %1$s_____________________驗</string>
-    <string name="username_label">試_Username_________________驗</string>
-    <string name="password_label">試_Password_________________驗</string>
-    <string name="login">試_Login___________驗</string>
+    <string name="server_trust_message">الشهادة المقدمة من %1$s غير موقعة من قبل مرجع مصدق موثوق.</string>
```
Comment (sorenoid) [inline toolkit/authentication/src/main/res/values-ar/strings.xml:18]: the format placeholder has been broken here and elsewhere in Arabic translations.

**Review by sorenoid — CHANGES_REQUESTED** [review]

**Comment by sorenoid** [comment]
@michi6571 @masa6674 Arabic translations appear broken as described in the diff.

**Comment by michi6571** [comment]
> @michi6571 @masa6674 Arabic translations appear broken as described in the diff.

@sorenoid Thank you for letting us know.  Please note that Git does NOT show RTL and BiDi correctly.  Could you please check the translation in the product and let us know if you still see the issue?  

cc @jona7150 @masa6674

File: toolkit/authentication/src/main/res/values-ar/strings.xml (hunk 1 above)
Comment (sorenoid) [reply toolkit/authentication/src/main/res/values-ar/strings.xml:18]: string looks ok when visually inspected in the IDE. The format placeholder is not broken into two pieces. weird.

**Review by sorenoid — APPROVED** [review]
Arabic format strings look ok in the IDE. thanks.

**Pull request merged**